kart/raster/import_.py

```diff
@@ -163,3 +164,10 @@ def existing_tile_matches_source(self, source_oid, existing_summary):
             source_oid = "sha256:" + source_oid
 
         return existing_summary.get("oid") == source_oid
+
+    def sidecar_files(self, source):
+        source = str(source)
+        if self.DATASET_CLASS.remove_tile_extension(source) != source:
+            sidecar_path = source + ".aux.xml"
```

Comment:
If GDAL has a tile `FOO.TIF` (uppercase), is the sidecar file `FOO.TIF.AUX.XML`, `FOO.TIF.aux.xml`, `FOO.tif.aux.xml`, or something else? Does GDAL care? Do we?

Reply:
Good point: GDAL is completely indifferent to case when it looks for sidecars. Updated Kart to behave the same way.